exonum/src/node/state.rs

```diff
@@ -564,6 +567,34 @@ impl State {
         ValidatorId(((height + round) % (self.validators().len() as u64)) as u16)
     }
 
+    /// Update known round for a validator, try reach actual round
+    /// returns new actual round
+    pub fn reach_actual_round(&mut self, id: ValidatorId, round: Round) -> Option<Round> {
```

Comment:
As far as I can see, this function doesn't actually "try to reach actual round", so I propose to rewrite the comment the following way:

Updates known round for a validator and returns a new actual round if byzantine majority of validators are on a higher round. Otherwise returns `None`.

Perhaps rename function too? Something like `get_actual_round`.